src/Microsoft.OpenApi.Readers/V3/OpenApiExampleDeserializer.cs

```diff
@@ -51,11 +52,19 @@ internal static partial class OpenApiV3Deserializer
         public static OpenApiExample LoadExample(ParseNode node)
         {
             var mapNode = node.CheckMapNode("example");
+            string description = null;
+            string summary = null;
 
             var pointer = mapNode.GetReferencePointer();
             if (pointer != null)
             {
-                return mapNode.GetReferencedObject<OpenApiExample>(ReferenceType.Example, pointer);
+                if (mapNode.Count() > 1)
```

Comment:
Because for a mapNode with a $ref pointer, it will always have a child property node(the $ref). The count() method is to check whether there are more property nodes besides the $ref i.e. looking for a summary and/or description node hence the greater than(<1) condition.

Reply:
would it be better to use any with a filter clause for $ref instead? to magic numbers usually come back to haunt you at some point by experience :)